test/powershell/Modules/Microsoft.PowerShell.Utility/PowerShellData.tests.ps1

```diff
@@ -1,83 +1,63 @@
-﻿Describe "Tests for the Import-PowerShellDataFile cmdlet" -Tags "Feature" {
+﻿Describe "Tests for the Import-PowerShellDataFile cmdlet" -Tags "CI" {
 
     It "Validates error on a missing path" {
 
-        $foundError = ""
         try
         {
             Import-PowerShellDataFile -Path /SomeMissingDirectory -ErrorAction Stop
+            Throw "Execution OK"
```

Comment:
I find this pattern confusing because of the message - a better message would be "execution should not have reached here" or something like that.